lib/pf/locationlog.pm

```diff
@@ -241,6 +243,9 @@ sub locationlog_db_prepare {
         FROM locationlog
         WHERE mac = ?
         ORDER BY start_time DESC LIMIT 1 ]);
+        
+    $locationlog_statements->{'locationlog_unique_ssids_sql'} = get_db_handle()->prepare(
+        qq [ SELECT DISTINCT `ssid` FROM `locationlog` WHERE `ssid` != '' AND `ssid` IS NOT NULL ORDER BY `ssid` ASC]);
```

Comment:
Check with James with the upcoming multi-tenant code if it's safe or will need to be adjusted.

Reply:
@extrafu is right, this will have to use the DAL, you can check with James whether this should be done before or after the merge of #2687

Reply:
Would need to be adjusted but it is not a big deal.

Reply:
@extrafu We'll ( @jrouzierinverse and I ) will address after DAL is merged.